src/stcal/resample/resample.py

```diff
@@ -202,15 +209,33 @@ def __init__(self, output_wcs, n_input_models=None, pixfrac=1.0,
                 At this time, output error array is not equivalent to
                 error propagation results.
 
+        include_var_in_err : list or None, optional
+            Indicates which variances to include in the ``err`` computation.
+            This field is used only if ``compute_err`` is ``from_var``.  If
+            None, all variances are included in the error computation.
+
         """
         # to see if setting up arrays and drizzle is needed
         self._finalized = False
         self._n_res_models = 0
 
         self._enable_ctx = enable_ctx
-        self._enable_var = enable_var
         self._compute_err = compute_err
 
+        if isinstance(enable_var, bool):
+            if enable_var:
+                self._enable_var = ['var_rnoise', 'var_poisson', 'var_flat']
```

Comment:
This is hardcoded and that is why I prefer class variables.

Reply:
I've now adopted your suggestion and do this via variance_array_names.